scripts/package_building/check_latest_tags.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+set -xe -o pipefail
+
+function resolve_tree_state {
+    repo_name="$1"
+    kernel_tree="$2"
+    
+    if [[ -d $repo_name ]];then
+        pushd "./${repo_name}"
+        git pull
+        popd
+    else
+        git clone $kernel_tree $repo_name
+    fi
+}
+
+function get_latest_commits {
+    repo_name="$1"
+    versions="$2"
+    results_path="$3"
+    
+    pushd "./${repo_name}"
+    full_tags="$(git tag -l --format='%(refname) %(taggerdate)' | grep "$(date +"%b %d")" | grep $(date +"%Y"))"
```

Comment:
can we substract one day?

it is safer to check all the tags from previous day, as the day has been completed